adserver/models.py

```diff
@@ -134,6 +135,28 @@ class Publisher(TimeStampedModel, IndestructibleModel):
         _("Allow house campaigns"), default=True
     )
 
+    # Payout information
+    payout_method = models.CharField(
+        max_length=100,
+        choices=PUBLISHER_PAYOUT_METHODS,
+        blank=True,
+        null=True,
+        default=None,
+    )
+    stripe_connected_account_id = models.CharField(
+        _("Stripe connected account ID"),
+        max_length=200,
+        blank=True,
+        null=True,
+        default=None,
+    )
+    open_collective_name = models.CharField(
+        _("Open Collective name"), max_length=200, blank=True, null=True, default=None
+    )
+    paypal_email = models.EmailField(
+        _("PayPal email address"), blank=True, null=True, default=None
+    )
+
```

Comment:
Should we have some kind of content block for `Other`? I'm thinking we should probably just not include it at this point, unless it's required for showing `None`?

Reply:
It isn't required although it is useful especially for showing the payout method on payout objects themselves.